src/test/java/eu/dissco/backend/service/MachineAnnotationServiceServiceTest.java

```diff
@@ -98,39 +100,51 @@ void testGetMassForObjectNoFilterMatch(List<Pair<String, List<String>>> filters)
   }
 
   @Test
-  void testScheduleMass() throws JsonProcessingException {
+  void testScheduleMass() throws JsonProcessingException, ConflictException {
     // Given
     var digitalSpecimen = givenDigitalSpecimenWrapper(ID);
     var masRecord = givenMasRecord(givenFiltersDigitalSpecimen());
     given(repository.getMasRecords(List.of(ID))).willReturn(List.of(masRecord));
-    given(masJobRecordService.createMasJobRecord(Set.of(masRecord), ID, ORCID, MjrTargetType.DIGITAL_SPECIMEN)).willReturn(
+    given(masJobRecordService.createMasJobRecord(Set.of(masRecord), ID, ORCID, MjrTargetType.DIGITAL_SPECIMEN, false)).willReturn(
```

Comment:
Might have missed it but is there a test which has batching on `true`?

Reply:
line 122: `testScheduleMassInvalidBatchRequest()`
Except for a validation check, batching being true/false doesn't affect the flow in this service (more the masJobRecord service)